tests/phpunit/tests/plugins/test-wp-importer.php

```diff
@@ -89,11 +93,31 @@ protected function _import_wp( $filename, $users = array(), $fetch_files = true
 	}
 
 	public function test_simple_import() {
-		$this->_import_wp( PLL_TEST_DATA_DIR . 'test-import.xml' );
-
+		$import_filepath      = PLL_TEST_DATA_DIR . 'test-import.xml';
+		$import_filepath_test = PLL_TEST_DATA_DIR . 'test-modified-import.xml';
+		$import_content  = strtr(
+			file_get_contents( $import_filepath ),
+			array(
+				'{lang_id_1}' => '<![CDATA[172]]>',
+				'{lang_id_2}' => '<![CDATA[174]]>',
```

Comment:
The `CDATA` being always present, I would keep it in the imported file and just have the numbers here.